### DISCUSSION TITLE
QuickNote Default display

### DISCUSSION THREAD

**Discussion by plegrand1** [gsantner/markor#2061] [Q&A] [answered]
Hello,
I've looked everywhere in the configuration options without success.
I would like the default display for "QuickNotes" to be read mode and not edit mode.
Is it possible to configure this option somewhere or is it not planned?
Thanks for your help.
Pascal

**Comment by plegrand1** [comment]
Any idea how to go about configuring this option if it exists?

**Reply by gsantner** [reply, marked as answer]
@plegrand1 Currently there is no such option. QuickNote / ToDo on the start page is set to edit at startup. If those would be viewing it wouldn't be really quick note taking anymore 😄 . However any other documents can be opened by a shortcut in view mode by default.
